README.md

```diff
@@ -6,12 +6,51 @@ Ionic demo world and resources
 Usage:
 
 ```
-cd Ionic
+git clone https://github.com/gazebosim/ionic_demo
+cd ionic_demo/ionic_demo/worlds
 gz sim -v 4 ionic.sdf
 ```
 
-
 In Harmonic:
 ```
 sudo apt install ros-rolling-ros-gz
 ```
+
+# Running the navigation demo
+
+![](../media/ionic-nav-demo.gif)
+
+This demo requires at least [ROS 2 Jazzy](https://docs.ros.org/en/jazzy/index.html).
+
+Create a new colcon workspace, install dependencies and build the packages,
+
+```
+mkdir -p ~/ionic_ws/src
+cd ~/ionic_w/src
+git clone https://github.com/gazebosim/ionic_demo
+
+source /opt/ros/jazzy/setup.bash
+rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
+
+cd ~/ionic_ws/
+colcon build
+```
+
+Launch the demo,
+
+```
+source ~/ionic_ws/install/setup.bash
+ros2 launch ionic_demo ionic_navigation_demo_launch.py headless:=0
+```
+
+On rviz, initialize the position at the origin towards the right of the map and start using the navigation commands provided on the rviz panels.
```

Comment:
can you provide more instructions / examples on how to initialize the position / navigation commands in rviz panel? Or maybe there's a nav2 doc page to shows how to do this?

Reply:
Gotcha, https://github.com/gazebosim/ionic_demo/pull/1/commits/46aef60c30f6fbda9123093b3dc635ce47e2a51d, I've added the gif and screenshots explicitly. Later on down the road we can use an orphaned media branch